spec/models/child_spec.rb

```diff
@@ -66,6 +67,11 @@
       expect(Child.approved_for_date(child.approvals.first.effective_on)).to include(child)
       expect(Child.approved_for_date(child.approvals.first.effective_on - 1.day)).to eq([])
```

Comment:
```suggestion
      expect(Child.approved_for_date(child.approvals.first.effective_on)).to not_include(inactive_child)
      expect(Child.approved_for_date(child.approvals.first.effective_on)).to not_include(deleted_child)
```

Reply:
Some thing here: inactive and deleted entries should still be present IMO.